jetty-server/src/main/config/etc/jetty-stats.xml

```diff
@@ -5,7 +5,9 @@
 <Configure id="Server" class="org.eclipse.jetty.server.Server">
   <Call name="insertHandler">
     <Arg>
-      <New id="StatsHandler" class="org.eclipse.jetty.server.handler.StatisticsHandler"></New>
+      <New id="StatsHandler" class="org.eclipse.jetty.server.handler.StatisticsHandler">
+        <Set name="asyncGraceful"><Property name="jetty.statistics.asyncGraceful" default="true"/></Set>
```

Comment:
"asyncGraceful" does not really convey the notion that is the shutdown that is graceful, and for full requests.
I would rename this to "gracefulShutdownWaitsForRequests".

Reply:
The graceful shutdown of waiting for requests isn't enabled by this setter, it is just switching the mode to whether we wait for async requests or not. Would `gracefulShutdownWaitsForAsyncRequests` be a better name?

Reply:
@lachlan-roberts it waits for requests, whether they are async or not, so no need for async in the name.

Reply:
`gracefulShutdownOnRequestsComplete` ?